<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>744</width>
    <height>412</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="MplWidget" name="MplWidget" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>431</width>
     <height>351</height>
    </rect>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget_Range">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>40</y>
     <width>201</width>
     <height>91</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <row>
    <property name="text">
     <string>Data Range (L)</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>Data Range (R)</string>
    </property>
   </row>
   <column>
    <property name="text">
     <string>Set Value</string>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="button_DataRange">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>150</y>
     <width>201</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Open Spanning</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_Done">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>290</y>
     <width>201</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Done!</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_DataRange_2">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>220</y>
     <width>201</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Crop Data</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MplWidget</class>
   <extends>QWidget</extends>
   <header>mplwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
